### ISSUE TITLE
ResNet101, OpenImages 5000 classes: Tensorflow -> ONNX

### ISSUE THREAD

**Issue by g7e** [Microsoft/MMdnn#594]
Platform (like ubuntu 16.04/win10): win10

Python version: 3.6.8

Source framework with version (like Tensorflow 1.4.1 with GPU): 1.13.0rc1 CPU

Destination framework with version (like CNTK 2.3 with GPU): ONNX

Pre-trained model path (webpath or webdisk path): https://storage.googleapis.com/openimages/2017_07/oidv2-resnet_v1_101.ckpt.tar.gz

Running scripts: 
`mmconvert --srcFramework tensorflow -in oidv2-resnet_v1_101.ckpt.meta --weights oidv2-resnet_v1_101.ckpt --dstFramework onnx -om resnet_v1_101.onnx --inNodeName map/TensorArrayStack/TensorArrayGatherV3 --dstNodeName resnet_v1_101/logits/BiasAdd --inputShape 299,299,3`

I'm trying to convert the tensorflow checkpoint that I mentioned above to ONNX.
I've followed the steps described by @JiahaoYao in the issue #206 and managed to generate the .onnx file. I inspected the file with Netron and, apparently, everything is ok - [1,299,299,3] as input and [1,1,5000] as output (I'd rather have [5000], though). However, when I load the model in either CNTK or ONNX, the output of the loaded model is [5000,1,10], resulting in invalid predictions. Can someone give it a try? Thanks.
